hydromt_fiat/fiat.py

```diff
@@ -759,6 +785,25 @@ def setup_building_footprint(
             building_footprint_fn,
             attribute_name,
         )
+        # Create BF folder in Exposure
+        building_footprints_exp = Path(self.root).joinpath("exposure" , "building_footprints")
+        if not os.path.exists(building_footprints_exp):
+            os.makedirs(building_footprints_exp)
+        if isinstance(building_footprint_fn,list):
+            for file in building_footprint_fn:
+                shutil.copy2(file, building_footprints_exp)
+        else:    
+            shutil.copy2(building_footprint_fn, building_footprints_exp)
+
+        # Create BF folder in Output
+        building_footprints_out = Path(self.root).joinpath("output" , "building_footprints")
+        if not os.path.exists(building_footprints_out):
+            os.makedirs(building_footprints_out)
+        if isinstance(building_footprint_fn,list):
+            for file in building_footprint_fn:
+                shutil.copy2(file, building_footprints_out)
+        else:    
+            shutil.copy2(building_footprint_fn, building_footprints_out)
```

Comment:
```suggestion
```
It is sufficient to only save the building footprints in the exposure/building_footprints folder